source/user/cc.rst

```diff
@@ -6,6 +6,10 @@ Coupled-cluster theory
 
 *Modules*: :mod:`cc`, :mod:`pbc.cc`
 
+The interfaces to coupled cluster and MP2 calculations look very similar
+in PySCF.  Therefore, note the similarities of this section to the
+:ref:`theory_mp2` documentation.
```

Comment:
You're repeating "similar" and "similarity". The latter could be replaced by "parallels"